doc/whats_new.rst

```diff
@@ -144,6 +188,13 @@ Enhancements
    - The :func: `ignore_warnings` now accept a category argument to ignore only
      the warnings of a specified type. By `Thierry Guillemot`_.
 
+   - The new ``results_`` attribute of :class:`model_selection.GridSearchCV`
+     (and similar utilities) can be easily imported into pandas as a
+     ``DataFrame``. Ref :ref:`model_selection_changes` for more information.
+     (`#6697 <https://github.com/scikit-learn/scikit-learn/pull/6697>`_) by
```

Comment:
By PR # do you mean adding the PR number or the hash?

If by PR # you mean PR number, it seems to be [the new convention](http://scikit-learn.org/dev/whats_new.html#new-features) as other entries (newer ones) seem to have it. I vaguely remember @amueller having an opinion about it.